src/i18n/en.json

```diff
@@ -822,5 +830,97 @@
+  },
+  {
+    "id": "getinvolved_software_list_dcrdata",
+    "translation": "<a href=\"https://github.com/decred/dcrdata\">dcrdata</a> - block explorer"
+  },
+  {
+    "id": "getinvolved_software_list_dcrweb",
+    "translation": "<a href=\"https://github.com/decred/dcrweb\">dcrweb</a> - Decred.org"
+  },
+  {
+    "id": "getinvolved_software_list_dcps",
+    "translation": "<a href=\"https://github.com/decred/dcps\">dcps</a> - Decred Change Proposals"
+  },
+  {
+    "id": "getinvolved_notsoftware_title",
+    "translation": "Not (primarily) Software"
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_dcrdocs",
+    "translation": "main documentation site at <a href=\"https://docs.decred.org/\">docs.decred.org</a>. The documents themselves live in folders corresponding to their sections within the <a href=\"https://github.com/decred/dcrdocs/tree/master/docs\">/docs/ folder</a>. The <a href=\"https://matrix.to/#/#documentation:decred.org\">#DCR documentation</a> channel is where people who work on dcrdocs discuss it. GitHub Issues are used to make suggestions about changes, like new pages or re-writing or re-structuring content. Pull Requests to change content should typically be in relation to some open issue and/or discussion in <a href=\"https://matrix.to/#/#documentation:decred.org\">#DCR documentation</a>."
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_dcrdesign",
+    "translation": " "
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_video-production",
+    "translation": "production of Decred video explainers, including scripts and storyboards for work in production. These videos go through stages of production and feedback is typically sought in <a href=\"https://matrix.to/#/#writers:decred.org\">#DCR Writers</a> at certain milestones."
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_decred-news",
+    "translation": "monthly <a href=\"https://xaur.github.io/decred-news/\">Decred Journal</a> maintained by @bee. Seeks review and contributions from other community members in <a href=\"https://matrix.to/#/#documentation:decred.org\">#DCR documentation</a> around the start of each month, with the journal being published around the 6th."
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_decred-events",
+    "translation": "@vj maintains a set of guidelines and best practices for attending events and hosting meetups. Community members should submit logs of events where Decred had a presence as well as information on upcoming events."
+  },
+  {
+    "id": "getinvolved_notsoftware_desc_richardred0x",
+    "translation": "has repositories for <a href=\"https://github.com/RichardRed0x/writing\">writing</a> (feedback usually sought before publication), <a href=\"https://github.com/RichardRed0x/analysis\">analysis</a> (data and scripts available, welcomes improvements or extensions to code), and an older repo with <a href=\"https://github.com/RichardRed0x/dash-analysis\">Dash data and analysis</a>."
```

Comment:
I would drop this bullet and replace it with new bullets, see comment on the PR for list of suggestions.

Reply:
I reworded this a little bit and then added the new bullets underneath it.